terraform/aws/inventory/ansible_inventory.tpl

```diff
@@ -1,6 +1,6 @@
 ${all_hosts}
 
-[faucet-web]
```

Comment:
Same question as before, why reverting this?

Reply:
https://github.com/dashpay/dash-networks-deploy/pull/10#discussion_r211407390

Reply:
👍